index.bs

```diff
@@ -1451,6 +1474,7 @@ an assertion. Its {{PublicKeyCredentialRequestOptions/challenge}} member must be
         unsigned long                        timeout;
         USVString                            rpId;
         sequence<PublicKeyCredentialDescriptor> allowCredentials = [];
+        UserVerificationRequirement          requireUserVerification = "preferred";
```

Comment:
we may have an impedance mis-match here with the default value of "preferred".  CTAP has UP default to `true` and UV default to `false` on its `authenticatorGetAssertion` method.

Reply:
These two are different things. CTAP `UV` is defining what to interpret if that parameter is not present in the API. Here is what the behavior should be via the web and how to handle 3 different use case for getAssertion. Its `UserVerificationRequirement` and not `UV`. 

We don't have to match these things.

Default is authenticator behavior (`preferred`)

Reply:
So is this a wontfix, then?

Reply:
Yes, IMO